dev/shiza16/Vehicle_Classifier.ipynb

```diff
@@ -0,0 +1,1355 @@
FZCCGEEEIIIYQQQgghRJWTxmchhBBCCCGEEEIIIYQQVU7GfBZCCCGEEEIIIYQQQohzscqYz5dCej4LIYQQQgghhBBCCCGEqHLS+CyEEEIIIYQQQgghhBBJrNcVAAAgAElEQVSiysmwG0IIIYQQQgghhBBCCHEuWobduBTS81kIIYQQQgghhBBCCCFElZPGZyGEEEIIIYQQQgghhBBVTobdEEIIIYQQQgghhBBCiHOxyrAbl0J6PgshhBBCCCGEEEIIIYSoctL4LIQQQgghhBBCCCGEEKLKybAbQgghhBBCCCGEEEIIcS4y7MYlkZ7PQgghhBBCCCGEEEIIIaqc9HwWopTNh84w5/vfsWoro7u1ZGK/9uViVu0+zoI1f4KCFnWDmH1LPwDe/OkPNh44DcBdAzoypENTp+b+d2w+lsCc1Xuwas3oDo2Z2LNluZhVf51hwcYDALQI82f2qG4cTMhg5spd5BQUYTQo/q9XK4a0jnB2+pds877jzFm6FqvVyuje7Zk49EqH5XOXrmXboRgA8guLSMvOY9ObU9l26BRzl/5SHHcyIZXZd15L/w4tnJr/P+WZma+zYfMfBAUG8N0n71V3OlVm85E45vy43Xacd27GxKuiysWs2nuKBev2AIoW4QHMHtObbccTmLtiZ3HMyZRMZt/Ym/6tGzgx+7/H2LwDHtfcAQYD5u1rKdrwXcVxUVfiecsjmOY/gTX2OCogBK+pb2JNiQPAevowhd+/78zU/5bNf51kzte/YrVqRveIYuLgrg7L5379K9uOnAEgv9BMWk4em+ZMLl6eYypg9Msf079dJNPG9HNq7n/HczMfI3pgb0ymfB6//3n27znosNyntjefL/9f8XR4vVC+/3IFLz3zKvUi6vLK288TFBxIRkYmj9zzDAnxSc4uwkXbfDCGOd/9ZtvX3VsxcUBHh+Vzv/+NbUdtx7FtX5vY9PIdANy78Ef2nEqiY5Nw3vm/YU7P/e/YfDKFub8exGrVjGoTwcSuTRyWL9sfyxubDhPq4wnATR0acF2bCLadTuPVXw8Vx51Mz2X2sHb0axbq1Pyryua9R3nls1W2+lufjky6prfD8vjUTJ7533dk5xVgtVp58IYB9GnXvJqy/Xs2HzjFnG822cp6ZWsmDuzssHzut5tKzmtFZtKyTWyafSdxaVk8smglFqsVs9XK2D7tuLFXm+oowiUxNIrCo+8Y2/fYvk2Yt69yWG5s3QOP3tejczMAKNq1Dsv+zSUBHp543jYdy9FdFK3/3Jmp/21dX7iV+v07YDEVsPmhhaTtO1kupsMTNxJ5Q288/H1Y0uL/iuf71Aum11t34+HnjTIY2DnrC2J/2e3E7C9M7b6dqP/cnWA0kPbFGpLf/cphufJwo8HrD+PVJhJLRjan7ptD0Zkk3CNCafnzfAqOxwKQ9+chYp+eD0DYo7cSeF0/jP612R81xulluli1ruxKwMP3oQwGcpf9RPZHSxyWe3RoR8BDU3Bv1pS0Z1/E9MsG23qdO+A/9d7iOPdGDUl95kXyN2zGVRhbd8bzxntAGSj6bSWFq790WO7e52rcrxoOViu6IJ+Cz97GmhADBiOe46diaBAJRiPmrWspXLW0mkpx8YzN2uMx9DbbeW3nOoo2Las4rnU3PMc8hGnh01jjjmNs2wv3XsOLlxvCGpK/4CmsCaeclboQF00an4Wws1itzPr2N967cyhh/j6Me2cZfVs3JDIssDjmVHImi9bt5sN7h+PnXYu0HBMAGw7EcCA2lS+mjqbIYmHSuz/Rq1UEtT09qqs4F8xi1cxauZv3bulNmJ8X4xato2/zukSG+BXHnErLYdFvh/jwtr74eXmQlpsPgJe7kRev7UKjoNokZZu45X+/0KNpKH4uUW4rs5as4b2pNxEW6Mu4WYvp264ZkfXqFMc8NmZA8eslv+zg4OlEALq2bMTSZ22NFpm5JkY8s5AerR1/8LuyUVcP4pbrr+WpF1+t7lSqjMVqZdbybbw3oT9hft6Me28lfVtFEBnqXxxzKjWLRRv28+Gdg/HzqkVaju0479o0nKVTrgYgM6+AEW8uo0ezutVSjkuiDHiMmET+By+is9LwnDwL84Ht6OQzjnEenrj3GIYl5rDDbJ2WQP5/HnNiwlXDYrUy68v1vDdlNGEBtRk393P6tm1KZN3g4pjHru9b/HrJr7s4eCbZYRvzfvydzs3qOy3nqhA9sBeNmzakf7eRdOjclhfmTuP6Ibc7xOTm5DGi39ji6e/XfsqqH20X1KbNmMq3X/zAN1/8QI8+XXn02ft59N5nnVqGi2WxWpn1zWbeu/sa2/f3m9/QN6oxkeEl39+PjexZ/HrJxn0cjE0pnr49uj35RWa+2nLAqXn/XRarZva6A7x7XWfCansybsnv9G0aQmRwbYe4IS3CebLfFQ7zujYI4ovxPQDIzC/i2g82cmWjYFyRxWpl5icrWPDIeMKC/Ljlhf8S3aElkfVDimPeX76RIV2jGNOvC8dik7nvzc9YMffBasz60lisVmZ9tYH3Jl9rO6+9/iV92zQhMjyoOOax0SUN70s27Ck+r4X4+bB46vV4uBnJKyjk+tmf07dNE0L9fZxejoumFB79xlLwzZvonHQ8x07DcnwPOi3eIcx8eHulDcvuPa7FeuaIM7KtUvX7t8evSTjf9X6EOp0i6T5rAitGTC8Xd2bNTg59sIZRmxzrbm0fHMnJ5Vs5/NFa/JvXY8DHj/HNlQ85KfsLZDBQ/4V7ODH+WYoSUmm27HWy1myl4Ojp4pCgMYOxZOZwKPpu/Ef0oe6TE4i5bw4AhacSOHJ1+f/n7LV/kLr4B1quX+C0olwyg4HAxx4k+f7HsCQlE/rhu5g2/ob5REljoiUxkfQXX6H2OMeG9IIdu0i69S4AlJ8vdb/6mIKt252a/t+iDHjeNIW8t59CZ6Tg/cRbmPdstTUu2xVtW0/Rxp8AMLbtTq3r78Q071ncOvUBN3fyXr4X3Gvh89wCiratR6
```

Comment:
It looks like this is running on the full dataset? It should run only on the training set.